hbase-metrics-api/src/main/java/org/apache/hadoop/hbase/metrics/Histogram.java

```diff
@@ -55,4 +55,10 @@
   @InterfaceAudience.Private
   Snapshot snapshot();
 
+  long[] getQuantiles(double[] quantiles);
```

Comment:
Do we have to change this interface? This is IA.LimitedPrivate.

Reply:
snapshot() will  be better, but the implementation will reset the data before, which makes no senses for histogram metrics. I haven't gotten the reason.Another thing is the interfaces in Histogram are not enough? There are getMin, getMax, usually we need quantiles?